ChangeLog.md

```diff
@@ -20,6 +20,10 @@ See docs/process.md for more on how version tagging works.
 
 2.0.27
 ------
+- Added some support for signal handling libc functions (raise, kill,
+  sigaction, sigpending, etc).  We still don't have a way to deliver signals from
+  the outside but these at least now work for sending signals to the current
+  thread (JS context) (#14883).
```

Comment:
This changelog note should probably be moved to 2.0.28 (since 2.0.27 was already released).

Reply:
Oops, I guess we never mark 2.0.27 as released: https://github.com/emscripten-core/emscripten/pull/14898